src/grandorgue/updater/GONewReleaseDialog.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
+
+#ifndef GONEWRELEASEDIALOG_H
+#define GONEWRELEASEDIALOG_H
+
+#include "config/GOConfig.h"
+#include "updater/GOUpdateChecker.h"
+#include <wx/wx.h>
+
+class GONewReleaseDialog : public wxDialog {
```

Comment:
Please use the dialogs/common/GOSimpledialog for the base class instead of wxDialog.

See an example in dialogs/GOOrganSettingsDialog as an example of how to add a custom button.